### PR TITLE
Adding team_aruba as maintainer for aruba modules

### PR DESCRIPTION
**Pull request by karthikeyan-dhandapani**
##### SUMMARY
Adding team_aruba as maintainer for the aruba modules.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
module and module_utils name: aruba
plugins, terminal and cliconf name: aruba.py

##### ADDITIONAL INFORMATION
Aruba would like to maintain the modules specific to Aruba Networks(HPE).

### CODE REVIEW

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -1585,6 +1589,7 @@ macros:
   team_aci: brunocalogero dagwieers devarshishah3 fadallar jmcgill298 koladiya mtorelli rsmeyers schunduri smnmtzgr
   team_aireos: jmighion
   team_aix: bcoca dagwieers d-little flynn1973 gforster kairoaraujo marvin-sinister mator molekuul MorrisA ramooncamacho wtcross
+  team_aruba: karthikeyan-dhandapani
```
Comment (Akasurde) [inline .github/BOTMETA.yml:1592]: @karthikeyan-dhandapani Are you missing jmighion in this team ?

**Review by Akasurde — CHANGES_REQUESTED** [review]

File: .github/BOTMETA.yml (hunk 1 above)
Comment (karthikeyan-dhandapani) [reply .github/BOTMETA.yml:1592]: Hi @Akasurde, he removed himself from the maintainer and assigned team_ansible with this pull request still open. #61469

File: .github/BOTMETA.yml [hunk 2]
```diff
@@ -286,7 +286,7 @@ files:
     maintainers: $team_aireos
     labels: [ aireos, cisco, networking ]
   $modules/network/aos/: dgarros jeremyschulman
-  $modules/network/aruba/: jmighion
+  $modules/network/aruba/: $team_aruba
```
Comment (Akasurde) [inline .github/BOTMETA.yml:289]:
```suggestion
  $modules/network/aruba/:
    maintainers: $team_aruba
    ignored: jmighion
```

File: .github/BOTMETA.yml [hunk 3]
```diff
@@ -734,7 +734,7 @@ files:
   $module_utils/network/aos:
     maintainers: dgarros jeremyschulman
   $module_utils/network/aruba:
-    maintainers: jmighion
+    maintainers: $team_aruba
```
Comment (Akasurde) [inline .github/BOTMETA.yml:737]:
```suggestion
    maintainers: $team_aruba
    ignored: jmighion
```

File: .github/BOTMETA.yml [hunk 4]
```diff
@@ -1585,6 +1589,7 @@ macros:
   team_aci: brunocalogero dagwieers devarshishah3 fadallar jmcgill298 koladiya mtorelli rsmeyers schunduri smnmtzgr
   team_aireos: jmighion
```
Comment (Akasurde) [inline .github/BOTMETA.yml:1590]:
```suggestion
   team_aireos: $team_ansible
```

File: .github/BOTMETA.yml (hunk 1 above)
Comment (Akasurde) [reply .github/BOTMETA.yml:1592]: Got it, thanks for the information. Could you please in-corporate changes from 61469 in this PR ? I will mark that PR as duplicate of this after update.

**Review by Akasurde — CHANGES_REQUESTED** [review]

**Comment by Akasurde** [comment]
@karthikeyan-dhandapani Thanks for taking up the responsibility as maintainer. Welcome.

**Pull request merged**